src/web/mjs/connectors/OmegaScans.mjs

```diff
@@ -1,13 +1,85 @@
-import WordPressMangastream from './templates/WordPressMangastream.mjs';
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
 
-export default class OmegaScans extends WordPressMangastream {
+export default class OmegaScans extends Connector {
 
     constructor() {
         super();
         super.id = 'omegascans';
         super.label = 'OmegaScans';
         this.tags = [ 'webtoon', 'scanlation', 'english', 'hentai'];
         this.url = 'https://omegascans.org';
-        this.path = '/manga/list-mode/';
+        this.api = 'https://api.omegascans.org';
+        this.nextInstance = 'Uzf9L765by7rm6wVbv5Sb';
+    }
+
+    async _initializeConnector() {
+        const uri = new URL(this.url);
+        const request = new Request(uri.href, this.requestOptions);
+        this.nextInstance = await Engine.Request.fetchUI(request, `__NEXT_DATA.buildId`);
+        console.log(`OmegaScans nextInstance is  ${this.nextInstance}`);

```

Comment:
Is this `log` left by intention or just forgotten to remove after finishing development/investigation?

Reply:
Its intended but can be removed if you want